### ISSUE TITLE
Connectome Wide Association Studies

### ISSUE THREAD

**Issue by mrjiaruiwang** [openconnectome/Statistical-Connectomics-Class#181]
So we saw in Genome Wide Association Studies (GWAS) that >80% of predictions did not hold up to biological validation. Now connectomics is a young field for which even the biology hasn't been figured out yet and I wonder how CWAS studies will do. Does anyone have a compelling argument for either case CWAS > GWAS or GWAS > CWAS or CWAS=GWAS?

**Comment by DSP137** [comment]
This may not actually answer your question, but I think it will be a combination of studies that will be more beneficial. We saw that in CWAS, there were limitations because we could determine associations, but they were rather broad. As stated in the Shehzad paper, CWAS "may decrease sensitivity to highly localized sets of connections," so this may be a start into determining regions of interest (ROI) to study further. I think it will be better to do a more compounded study where we use perhaps CWAS to determine ROIs and then study the ROIs for more detail.

**Comment by akim1** [comment]
Genome is a genotype. Genomes are essentially the most basic form of biological information, and in that respect, they're not an accurate picture of what is actually going on in the cell. The reason for this is that you lose a lot of important information such as post-translational modifications, differential expressions in different tissues, etc. which aren't really part of the genomic information, per se. But they do give really good information for Mendelian defects where basically genotype is phenotype.

Connectome is a phenotype. If the disease is a direct manifestation of a defect in the connectome, then it makes sense to measure the connectome, and the connectome should directly reveal information. So my guess is CWAS >>> GWAS.

**Comment by SandyaS72** [comment]
Maybe this is a dumb question, but doesn't this heavily depend on the accuracy and quality of the connectome you get for each person? What assumptions are you making about that?

**Comment by akim1** [comment]
You're right. But if the measurement is bad, then there's very little useful things you could get out of the connectome anyway.

**Comment by gkiar** [comment]
But what if they're "consistently" bad? For instance, we know that we aren't properly mapping the brain for any number of reasons, but our errors are consistent across subjects. Then, since all we're doing is comparing sets with equal error, don't the errors kind of disappear?

**Comment by akim1** [comment]
Well... I was trying to explain the reason why GWAS don't hold up to 80% of biological validation. That seems to suggest that the genome doesn't actually contain the information that we want.

At least, if the disease is from the connectome, we know that the information is in the connectome somewhere regardless of how dismal the quality of the measurement might be. An awful correlation seems to be much more desirable than no correlation, especially if intuition/biology tells us that it is highly likely a correlation would exist.

**Comment by itsthefedora** [comment]
Um ... Do we know these diseases are "from the connectome"? I think that's entirely conjecture at this point.

Greg, if you make the same errors for everyone, they definitely don't just "disappear". Trivially, if you zero out a bunch of connections for every subject, you will not find an association between those connections and some phenotype, even if those connections are actually implicated.

Jerry, I think that the primary concerns for GWAS were statistical: basically, I would posit that 80% of conclusions from GWAS ended up being wrong because those original studies didn't correct for multiple comparisons sufficiently well (for example, using the Benjamini-Hochberg algorithm for controlling the FDR in cases where the dependency structure of the genes means that BH doesn't actually control the FDR). I bet if you went through the entire GWAS literature and Bonferroni'd every single hypothesis, you'd probably dramatically reduce the false positive rate.

So, CWAS >,=,< GWAS is entirely dependent on whether people do the math right this time around. Given this, I would probably predict CWAS <= GWAS.
